3Box/3box-profile-hover.md

```diff
@@ -0,0 +1,210 @@
+profiles in a smooth way adding the following lines:
+
+```
+.ethAddress {
+        padding: 20px 0px 0px 40px;
+        height: 80px;
+}
+```
+
+Now you can run the React app.
+
+`npm start`
+
+![App](https://github.com/Solexplorer/Content/blob/master/3Box/images/app.png)
+
+
+# Using the HTML element
+
+In order to use the profile hover with a HTML app, we need to add the script at
+the end of the page.
+
+``<script type="text/javascript" src="https://unpkg.com/profile-hover"></script>``
+
+To display and address, you just need to add the following tag.
+
+` <threebox-address data-address='0x442dccee68425828c106a3662014b4f131e3bd9b'></threebox-address>`
+
+There are few other ways to customize your profile hover, you can add `data-display='full'`
+to show the entire address instead of the shortened one.
+
+You can also decide to not use the default styling using `data-theme='none'`.
+
+This is an example of how you can use profile hovers in a HTML App.
+
+```html
+<html>
+  <head>
+    <title>Profile demo</title>
+    <style>
+      .ethAddress {
+        padding: 20px 0px 0px 40px;
+        height: 80px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class='ethAddress'>
+      <threebox-address data-address='0x442dccee68425828c106a3662014b4f131e3bd9b' data-display='full'></threebox-address>
+    </div>
+    <div class='ethAddress'>
+      <threebox-address data-address='0xfc27bf5bdc304ea27e74fd3f514e5400f0a9e76d' data-theme='none'>
+      </threebox-address>
+    </div>
+  </body>
+    <script type="text/javascript" src="../dist/widget.js"></script>
+  </html>
+```
+
+## Customize the profile hover
+
+Imagine if you like the hover but you want to modify how they are visualized,
+this is in fact possible. Let's take a look at the important properties.
+
+The `address` property is required in order to display the profile hover since every 3Box
+profile is associated with an address.
+
+The `showName` property makes it possible to show the user's name from their 3Box profile,
+if you don't use the property it shows the address shortened. If a user didn't add a
+name to the profile, it will show the shortened address.
+
+The `url` property allows redirecting to a specific link when clicking on the `Tile`.
+
+The `displayFull` property shows the entire address instead of the shortened address.
+Remember that if you add the `showName` property after, it will override it (look
+at the penultimate profile in the example above ).
+
+These are the most important properties, if you want to take a look at all of them,
+check them out [here](https://github.com/3box/profile-hover#prop-types).
+
+## Conclusion
+
+We have seen how to use this feature, now I hope you will use it to improve your
+UI. 
+If you have any questions regarding profile hovers head over to the official
+documentation here: https://docs.3box.io/ and if you want to discuss something
+there is an apposite [Discord](https://discordapp.com/invite/TAefehN) for that.
```

Comment:
What's an "apposite"?

Reply:
It's a synonim of appropriate. See here https://dictionary.cambridge.org/dictionary/english/apposite